### PR TITLE
Avoid requiring jQuery for sending the coverage report back.

### PR DESCRIPTION
**Pull request by rwjblue**
Works the same:

screenshot

And doesn't troll apps running without jQuery (don't ask yet if that is
a thing plz :wink: ).

### CODE REVIEW

**Comment by rwjblue** [comment]
tests are failing because `ember test` is timing out, I'm not sure why yet.

When using this `npm link`'ed into a test ember-cli project it works properly...

**Comment by kategengler** [comment]
It looks like Phantom isn't happy with this. I have no patience for that so I went ahead and got rid of Phantom from the testem config and added some more output in the case of failure so future failures will be less mysterious (https://github.com/kategengler/ember-cli-code-coverage/commit/dc9c95f2f0a8c80488e8e198cc7f01fbb4040a24)

File: lib/templates/test-body-footer.html [hunk 1]
```diff
@@ -3,37 +3,35 @@
...
+          return name + ' ' + data.total[name.toLowerCase()].pct + '%'
+        });
+
+      var resultsText = document.createTextNode(results.join(' | '));
+      var element = document.createElement('div');
+      element.style = 'background-color: white; color: black; border: 2px solid black; padding: 1em;position: fixed; left: 15px; bottom: 15px';
+      element.appendChild(resultsText);
+      document.body.appendChild(element);
+
+      callback();
```
Comment (kategengler) [inline lib/templates/test-body-footer.html]: This was wrapped in an `if (callback)` before, for the case when it is run at `/tests` and there is no Testem callback. Not sure how to write a test that could catch that.

Comment (rwjblue) [inline lib/templates/test-body-footer.html]: Gotcha.  I will update, but I'm also unsure of how to test that since we can't run `ember test`...

Comment (kategengler) [inline lib/templates/test-body-footer.html]: Yeah, I guess so long as it's there...

**Pull request merged**